src/microbe_stage/Microbe.cs

```diff
@@ -1426,23 +1426,61 @@ private void OnIGotAddedToColony()
         if (ColonyParent == null)
             return;
 
-        var oldRotation = Rotation;
-        var vectorToParent = GlobalTransform.origin - ColonyParent.GlobalTransform.origin;
+        var newTransform = GetNewRelativeTransform();
+
+        Rotation = newTransform.rotation;
+        Translation = newTransform.translation;
+
         ChangeNodeParent(ColonyParent);
+    }
+
+    /// <summary>
+    ///   This method calculates the relative rotation and translation this microbe should have to his microbe parent.
```

Comment:
Would it be possible to reword this so that the microbe isn't referred to by a gendered pronoun. I've been careful to not do this elsewhere when commenting so this looks odd to me.